backend/data/processors/conditions_preprocessing.py

```diff
@@ -34,6 +34,11 @@ def preprocess_conditions():
         conditions["original"] = original
         processed = original
 
+
+        processed = remove_extraneous_comm_data(conditions, processed)[1]
+        if remove_extraneous_comm_data(conditions, original)[0] != "":
+            conditions["handbook note"] = remove_extraneous_comm_data(conditions, original)[0]
```

Comment:
afaik this should be handbook_note, with an underscore, not a space.

Reply:
also:
1. is the second argument mean to change from being `processed` in the first one to `original` in the next two?
2. consider only making one call in lines 39 and 40 and just storing the result in a variable, also this way you can name that variable to better communicate the intention of this bit of code